prime_ir/Dialect/Ring/Python/dialects/ring.py

```
@@ -0,0 +1,2 @@
+from .._mlir_libs._ring import *
```

Ruff (0.16.1)

[error] `from .._mlir_libs._ring import *` used; unable to detect undefined names

(F403)